### ISSUE TITLE
Error with infiniteLoop

### ISSUE THREAD

**Issue by SultanSaribas** [MarcinusX/NumberPicker#148]
When I set infiniteLoop to true such `infiniteLoop: true`, it throws me an error such as below. I couldn't find any information about the usage of this parameter. I saw, in the NumberPicker class, it is declared as ScrollController, but it is returned as InfiniteScrollController. I tried to change it, but still it doesn't work.
The following _TypeError was thrown building NumberPicker(state: _NumberPickerState#7c94f):
type 'ScrollController' is not a subtype of type 'InfiniteScrollController' in type cast

**Comment by ashellz** [comment]
Have you fixed this issue?

**Comment by SultanSaribas** [comment]
> Have you fixed this issue?

No, it seemed it was related to the package code itself, so I switched to another package called “ carousel_slider”.
